plugins/woocommerce-admin/client/analytics/report/orders/table.js

```diff
@@ -199,7 +199,8 @@ class OrdersReportTable extends Component {
 							: [],
 						formattedProducts.map( ( product ) => ( {
 							label: sprintf(
-								__( '%s× %s', 'woocommerce' ),
+								/* translators: 1: quantity, 2: product name */
```

Comment:
Interesting choice here; only the "x" would be translated since both of these are variables. I wonder if the original label really needs translation?

Reply:
I didn't quite notice that! Sharp eyes! But I'll leave it as it is 😆